src/app/api/routineApi.ts

```diff
@@ -56,3 +56,52 @@ export const fetchRoutines = async (query = ''): Promise<Routine[]> => {
 
   return filteredRoutines;
 };
+
+export const fetchRoutineDetails = async (routineId: string): Promise<Routine> => {
+  await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
I dont think we need these timeouts. Let our application be fast 😎

Reply:
These are just mocked functions which should be replaced with real api calls. It's the scope of another task in jira.

Reply:
Yes, you are right, I see it